app/models/spotlight/resources/iiif_manifest.rb

```diff
@@ -12,20 +12,19 @@ class IiifManifest
       def add_thumbnail_url
         return unless thumbnail_field
 
-        if manifest['thumbnail'].present?
-          if manifest&.[]('thumbnail')&.[]('service')&.[]('profile')&.match? 'api/image'
-            solr_hash[:thumbnail_square_url_ssm] =
-              "#{manifest&.[]('thumbnail')&.[]('service')&.[]('@id')}/full/100,100/0/default.jpg"
-          end
-          solr_hash[thumbnail_field] = manifest['thumbnail']['@id']
-        else
-          first_image_id = manifest&.[]('sequences')&.[](0)&.[]('canvases')&.[](0)
-            &.[]('images')&.[](0)&.[]('resource')&.[]('service')&.[]('@id')
-          if first_image_id
-            solr_hash[:thumbnail_square_url_ssm] = "#{first_image_id}/full/100,100/0/default.jpg"
-            solr_hash[thumbnail_field] = "#{first_image_id}/full/!400,400/0/default.jpg"
-          end
+        image = manifest['thumbnail'] if manifest['thumbnail'].present?
+        image ||= manifest.dig('sequences', 0, 'canvases', 0, 'images', 0, 'resource')
```

Comment:
I'm getting an `undefined method `dig' for #<IIIF::Presentation::Manifest:0x00007fc380ed3028>` when trying to index https://cdm15808.contentdm.oclc.org/iiif/info/p15808coll19/518/manifest.json . Maybe we need a better test here?

Reply:
Maybe so. I guess that's why we had the... interesting.. alternative.